bindings/dart/example/assets/call_data/example.json

```diff
@@ -1,268 +1,788 @@
+      "day": 4,
+      "url": "url://dom_3.test/0",
+      "domain": "dom_3.test",
+      "rank": 4,
+      "user_action": 1
+    },
+    {
+      "id": "1fc20ed5-b70b-4147-b911-e98752872d67",
+      "relevance": 0,
+      "user_feedback": 2,
+      "session": "32b42d89-ba38-47a0-8721-573a63402fef",
+      "query_count": 0,
+      "query_id": "e04dd090-2b57-4100-befa-4fa050560a06",
+      "query_words": "birds",
+      "day": 4,
+      "url": "url://dom_4.test/0",
+      "domain": "dom_4.test",
+      "rank": 5,
+      "user_action": 1
+    },
+    {
+      "id": "725b6dc2-7db2-452c-84cb-1b2ea14f8b5b",
+      "relevance": 0,
+      "user_feedback": 2,
+      "session": "32b42d89-ba38-47a0-8721-573a63402fef",
+      "query_count": 0,
+      "query_id": "e04dd090-2b57-4100-befa-4fa050560a06",
+      "query_words": "birds",
+      "day": 4,
+      "url": "url://dom_3.test/1",
+      "domain": "dom_3.test",
+      "rank": 6,
+      "user_action": 1
+    },
+    {
+      "id": "c7923fa0-aac2-4683-b523-d6d0995e76a5",
+      "relevance": 0,
+      "user_feedback": 2,
+      "session": "32b42d89-ba38-47a0-8721-573a63402fef",
+      "query_count": 0,
+      "query_id": "e04dd090-2b57-4100-befa-4fa050560a06",
+      "query_words": "birds",
+      "day": 4,
+      "url": "url://dom_2.test/1",
+      "domain": "dom_2.test",
+      "rank": 7,
+      "user_action": 1
+    },
+    {
+      "id": "1ccc0b33-ffbf-43c2-89f9-b0288cd7725f",
+      "relevance": 0,
+      "user_feedback": 1,
+      "session": "32b42d89-ba38-47a0-8721-573a63402fef",
+      "query_count": 0,
+      "query_id": "e04dd090-2b57-4100-befa-4fa050560a06",
+      "query_words": "birds",
+      "day": 4,
+      "url": "url://dom_5.test/0",
+      "domain": "dom_5.test",
+      "rank": 8,
+      "user_action": 1
+    },
+    {
+      "id": "7e81cc2b-da81-496e-8576-9aa2bd0ba0b7",
+      "relevance": 0,
+      "user_feedback": 2,
+      "session": "32b42d89-ba38-47a0-8721-573a63402fef",
+      "query_count": 0,
+      "query_id": "e04dd090-2b57-4100-befa-4fa050560a06",
+      "query_words": "birds",
+      "day": 4,
+      "url": "url://dom_4.test/1",
+      "domain": "dom_4.test",
+      "rank": 9,
+      "user_action": 1
+    },
+    {
+      "id": "7e5cd897-1f4a-4ada-914b-67a36449f89d",
+      "relevance": 0,
+      "user_feedback": 2,
+      "session": "32b42d89-ba38-47a0-8721-573a63402fef",
+      "query_count": 0,
+      "query_id": "e04dd090-2b57-4100-befa-4fa050560a06",
+      "query_words": "birds",
+      "day": 4,
+      "url": "url://dom_6.test/0",
+      "domain": "dom_6.test",
+      "rank": 10,
+      "user_action": 1
+    },
+    {
+      "id": "d1c41b13-435d-4e37-938a-9302438898dd",
+      "relevance": 0,
+      "user_feedback": 2,
+      "session": "32b42d89-ba38-47a0-8721-573a63402fef",
+      "query_count": 0,
+      "query_id": "e04dd090-2b57-4100-befa-4fa050560a06",
+      "query_words": "birds",
+      "day": 4,bindings/dart/example/assets/call_data/example.json
```

Comment:
this should not belong here :)

Reply:
Wtf :scream_cat: 

Not running tests can really bite you, even when just regenerating some files :wink: